### PR TITLE
Add property key to enable/disable services integration

### PR DESCRIPTION
**Pull request by eddumelendez**

### CODE REVIEW

**Comment by eddumelendez** [comment]
it's pending to add tests

**Comment by maciejwalkowiak** [comment]
Looks good! I think in the future we would add there properties to corresponding `@ConfigurationProperties` classes as there is a plan anyway to add ability to configure custom endpoint - so that they would go away from `additional-configuration-properties`, but this can be done once we get there.

**Comment by eddumelendez** [comment]
most of the properties that are declared in `@ConditionalOnProperty` are not declared in `@ConfigurationProperties` because they are not use anywhere. Are you planning to used these properties as part of the logic?

File: spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextStackAutoConfiguration.java [hunk 1]
```diff
@@ -51,6 +51,7 @@
 		ContextDefaultConfigurationRegistrar.class })
 @ConditionalOnClass(name = "com.amazonaws.services.cloudformation.AmazonCloudFormation")
 @EnableConfigurationProperties(AwsStackProperties.class)
+@ConditionalOnProperty(name = "cloud.aws.stack.enabled", havingValue = "true")
```
Comment (maciejwalkowiak) [inline spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextStackAutoConfiguration.java:54]: This is one way we can tackle disabling Cloudformation but I am wondering why to even add Cloudformation dependency in first place. 

Wouldn't it make more sense if we mark Cloudformation dependency as optional and enable configuration if user adds it to the classpath? I am thinking about similar approach to other integrations provided in context module.

Comment (eddumelendez) [reply spring-cloud-aws-autoconfigure/src/main/java/org/springframework/cloud/aws/autoconfigure/context/ContextStackAutoConfiguration.java:54]: agree on that. I will remove this commit in order to be address the issue later then.

**Pull request merged**